.meta.cfg

```diff
@@ -0,0 +1,7 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+[meta]
+template = pure-python
+commit-id = 05894360cac01f1027889dd14a23ae372e808b3f
+fail-under = 0
```

Comment:
Could you please set the value, that was previously set in tox.ini (86) if that is still valid?

Reply:
No, because this was never run before, and test coverage is way below that number.